README.md

```diff
@@ -86,27 +86,38 @@ oc apply -f examples/secret.yaml
 oc extract secret/off-cluster-token -n openshift-console --to ./examples --confirm
 ```
 
+If you have ran `source ./contrib/oc-environment.sh` before, unset all environment variables by running the following command:
```

Comment:
so I would probably suggest to unset these at the begining of the `oc-environment.sh`